### ISSUE TITLE
Make components easy for internalization i18n

### ISSUE THREAD

**Issue by rexolion** [union-platform/ui-kit#22]
# Clarification and motivation

Union is an international project, so ui-kit should be too

# Acceptance criteria

All the component in-built aria-labels and others should be entered in API, and English as default for backward compatibility.

**Comment by rexolion** [comment]
Fixed in #31!
